test/conftest.py

```diff
@@ -404,8 +405,9 @@ def pudl_settings_dict(request, pudl_input_output_dirs):  # noqa: C901
 
 @pytest.fixture(scope="session")  # noqa: C901
 def ferc1_dbf_datastore_fixture(pudl_datastore_fixture):
```

Comment:
Is this fixture still required, now that we're constructing the `Ferc1DbfReader` inside of `test_ferc1_schema`?

Reply:
I thought I have removed this, but apparently it was left behind. Now it's gone again.

Reply:
It looks like it's still here. Is there a commit that still needs to be pushed?